earth_enterprise/src/installer/common.sh

```diff
@@ -80,6 +80,26 @@ xml_file_get_xpath()
         tail -n +2 | head -n -1
 }
 
+is_package_installed()
+{
+    local package_installed_retval=1
```

Comment:
Q: Is there a reason to use a variable, rather than just return the value?

Reply:
Not really. I saw a couple of nearby functions do it that way, so used that style.  Looking again I see later in the same file several functions just return.

Reply:
It's a style preference, but I think it's less code and reads better to just return, if you can.